doc/api/next_api_changes/2018-02-05-AL.rst

```diff
@@ -0,0 +1,8 @@
+Changes to the internal tick handling API
+`````````````````````````````````````````
+
+``Axis.iter_ticks`` (which only served as a helper to the private
+``Axis._update_ticks``) is deprecated.
+
+The signature of the (private) ``Axis._update_ticks`` has been changed to not
```

Comment:
`axis_artist.AxisArtist._update_ticks()` also takes a `renderer` argument; and it uses it. Not quite clear if we want to keep the APIs similar in both worlds.

Reply:
They don't even return the same kind of things (one returns a list of ticks, the other returns a bounding box) so I wouldn't worry about that.

Reply:
In that case, I approve.